### PR TITLE
docs: add mgmt extension

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation

This PR is derived from dev-submitted PR #100 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mbshields** [comment]
In the list of functionality currently supported by the mgmt extension, added ability of users to upload certificate and public key files.

Also, fixed broken links in Configuring zot and in the Using Kind article.

**Review by peusebiu — APPROVED** [review]
lgtm

File: docs/general/extensions.md [hunk 1]
```diff
@@ -65,9 +66,18 @@ The following extensions are currently supported by zot:
...
 
-For information about configuring zot extensions, see [*Configuring zot*](../admin-guide/admin-configuration.md).
+### **Mgmt**
+
+The **mgmt** extension provides an API endpoint for configuration management.
+
+The functions currently implemented by this extension include:
+
+- Reports which current authentication methods are enabled in the zot configuration.  This function can be used by all users, authenticated or unauthenticated.
+- Allows users to upload certificates and public keys for verifying images.
```
Comment (rchincha) [inline docs/general/extensions.md:81]: This line we changing functionality/url ... @andaaron

File: docs/admin-guide/admin-configuration.md [hunk 2]
```diff
@@ -359,6 +361,62 @@ The following table lists the configurable attributes of the `lint` extension.
...
 
+<a name="mgmt_config"></a>
+
+## Configuration management
+
+The `mgmt` extension provides an API endpoint for configuration management.
+
+To configure `mgmt`, add the `mgmt` attribute under `extensions` in the configuration file, as shown in the following example:
+
+```json
```
Comment (andaaron) [inline docs/admin-guide/admin-configuration.md:372]: We are also planning to remove this, and enable the URL if both search and UI are enabled. We are also not sure we want to advertise the URL for getting the configuration.

**Pull request closed without merging**